templates/includes/sidebar.html

```diff
@@ -1,4 +1,12 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: {{ user.username }}</li>
```

Comment:
You need to add a link to the user's detail page on the username (readme task 3)